sci-mathematics/calc/calc-2.12.5.3.recipe

```diff
@@ -0,0 +1,107 @@
+SUMMARY="C-style arbitrary precision calculator"
+DESCRIPTION="Calc is an interactive calculator which provides for \
+easy large numeric calculations, but which also can be easily programmed for \
+difficult or long calculations. It can accept a command line \
+argument, in which case it executes that single command and exits. \
+Otherwise, it enters interactive mode. In this mode, it accepts \
+commands one at a time, processes them, and displays the answers. \
+In the simplest case, commands are simply expressions which are evaluated."
+HOMEPAGE="http://www.isthe.com/chongo/tech/comp/calc/"
+COPYRIGHT="1999-2014  Landon Curt Noll"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="http://www.isthe.com/chongo/src/calc/calc-$portVersion.tar.bz2"
+CHECKSUM_SHA256="0fcf60917efb10aab145f1180c278a3ab63d789aad6d5e5f066a4458b5f9b692"
+PATCHES="calc-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
```

Comment:
Good news: it builds fine on x86_gcc2 primary arch!
This means you can (and even should) remove the **`!`** in **`!x86_gcc2`** in both ARCHITECTURES and SECONDARY_ARCHITECTURES ;-)
```
ARCHITECTURES="x86_gcc2 x86 x86_64"
SECONDARY_ARCHITECTURES="x86_gcc2 x86"
```